packages/broker/src/plugins/storage/BucketManager.ts

```diff
@@ -1,15 +1,14 @@
 import { Client } from 'cassandra-driver'
 import Heap from 'heap'
 import { types as cassandraTypes } from 'cassandra-driver'
-import { Logger } from 'streamr-network'
+import { Logger, Protocol } from 'streamr-network'
 import { Bucket, BucketId } from './Bucket'
+import { SPIDKey } from '../../../../network/node_modules/streamr-client-protocol/dist/src'
 const { TimeUuid } = cassandraTypes
 
 const logger = new Logger(module)
 
-type StreamPartKey = string
-
-interface StreamPartState {
+interface SPIDState {
```

Comment:
Yeah, now when we see all the use cases of `SPID`s, it would be good to time to check the big picture. I totally agree with @harbu that especially the class names (e.g. `SPIDState` and `SPIDState`) are misleading.

If we find a new self-explanatory term, that would be great. On the other hand, the term stream partition is already used in end user UIs and documentation, and it is easy to understand as it is  plain English. 

To me, `SPID` feels like an independent term, not just an abbreviation of a stream partition. Therefore in some sense, we have now two words for the same concept: `SPID` and `stream partition`. Maybe it is just my interpretation. But if that's a more common interpretation, then there's some unnecessary complexity which we could maybe optimize?

When we introduced the `SPID` class few months ago, we didn't yet include the `SPIDKey` type. Now when we have both, it sounds to me that `SPIDKey` represents the ID of a stream partition, and `SPID` is the  actual stream partition. How about if we rename `SPID` to `StreamPartition` and `SPIDKey` to `SPID`/`StreamPartitionID`? 

That solution would make the method parameter types very guessable. An API method `addStreamPartition(streamPartition: StreamPartition)` would be super easy to use.